lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -78,7 +78,7 @@ def is_api_available(self) -> bool:
         try:
             self._client.sys.read_health_status(standby_ok=True)
             return True
-        except (VaultError, RequestException) as e:
+        except (VaultError, RequestException, OSError) as e:
```

Comment:
A change to this file should come with a bump to its version

Reply:
No need for this change anymore, so undone.